src/bin/test.rs

```diff
@@ -26,7 +26,7 @@ pub const USAGE: &'static str = "
 Execute all unit and integration tests of a local package
 
 Usage:
-    cargo test [options] [--] [<args>...]
+    cargo test [options] [-p SPEC --package SPEC]... [--] [<args>...]
```

Comment:
@alexcrichton Thanks for the reminder! I have a candidate implementation working in `0.6.69`. See my proposal for details: https://github.com/docopt/docopt/issues/275 (Even though it's working in the Rust implementation, I probably wouldn't add it into Cargo quite yet in case the proposal changes. If it's ultimately rejected, I might still leave it in because I think it's really really useful.)